client/src/data/types/donationForm.ts

```diff
@@ -8,12 +8,40 @@ export enum ItemCondition {
     POOR = "POOR"
 }
 
-export default interface DonationForm {
+export enum ItemStatus {
+    PENDING_APPROVAL = "PENDING_APPROVAL",
+    PENDING_DROPOFF = "PENDING_DROPOFF",
+    PENDING_MATCH = "PENDING_MATCH",
+    MATCHED = "MATCHED"
+}
+
+export interface DonationFormContributionTuple {
```

Comment:
nit: put `DonationFormContributionTuple` in `client/src/data/types/requestGroup.ts`

Reply:
oh how come it should go in `requestGroup.ts`?